index-cheminformatics.md

```diff
@@ -64,6 +83,11 @@ To orchestrate tools and help users with their analyses, several workflows are a
 
 The workflows are available in the [Shared Workflows](https://cheminformatics.usegalaxy.eu/workflows/list_published), with the label "***cheminformatics***".
 
+Workflow | Description
+--- | ---
+[GROMACS](https://cheminformatics.usegalaxy.eu/u/simonbray/w/molecular-dynamics-1) | Molecular dynamics simulation with GROMACS
```

Comment:
Can we give that one a better name, e.g. remove the -1. Ideally this workflow should stay forever and should be easily shareable.